content/1.7/back-office/login.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Login into Back-Office
+weight: 1
+---
+# LOGIN
+
+![Login UI](static/login.png)
+
+This page allows any authorized user to connect to the PrestaShop back-office. 
+
+## Top UI Section
+
+- The top section represents the Prestashop main title name and the Prestashop version, that the administrator is accessing.
+- Prestashop version - 1.7.8.0.
```

Comment:
Is the spec dedicted only for 1.7.8.0 version?

Reply:
@marwachelly I am not really sure, @MatShir told earlier to define the version, because we are kick-offing specifications from 1.7.8.0 from last year :)